.github/workflows/src/summarize-checks/summarize-checks.js

```diff
@@ -42,9 +41,7 @@ import {
   typeSpecRequirementDataPlaneTsg,
 } from "./tsgs.js";
 
-import fs from "fs/promises";
-import os from "os";
-import path from "path";
+import { unzipSync } from "zlib";
```

Comment:
And this is actually present? Nice.

Reply:
yep builtin: https://nodejs.org/docs/latest-v20.x/api/zlib.html